### ISSUE TITLE
Missing tabindex="0" on responsive tables

### ISSUE THREAD

**Issue by hannahiss** [Orange-OpenSource/Orange-Boosted-Bootstrap#1711]
### Prerequisites

- [X] I have [searched](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

In [responsive tables](https://boosted.orange.com/docs/5.2/content/tables/#responsive-tables) paragraph, on the first example, a `tabindex="0"` is present ; it seems it has been added for accessibility purposes on Internet Explorer and Safari, to allow users to focus into tables and so use the browser lift with keyboard only. But, if Internet Explorer is no longer supported, Safari is... 

`tabindex="0"` should be added on all examples with table-responsive, and be documented.

### Reduced test cases

Go to https://boosted.orange.com/docs/5.2/content/tables/#responsive-tables on Safari on small screen and try to use the lift under the responsive tables.

### What operating system(s) are you seeing the problem on?

macOS, iOS

### What browser(s) are you seeing the problem on?

Safari

### What version of Boosted are you using?

v5.2

**Comment by julien-deramond** [comment]
This issue was confirmed in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2162, where `axe-core` dep has been bumped after regeneration of our `package-lock.json`, leading to changes in pa11y results.

More information at https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2162#issuecomment-1649653748.

One of the new errors caught by pa11y is that we should add `tabindex="0"` whenever `.table-responsive*`s are used for a11y reasons.

A fix has been proposed upstream via https://github.com/twbs/bootstrap/pull/38971.

⚠️ **This issue is on hold until we have feedback from the upstream PR.**

**Comment by MaxLardenois** [comment]
Check if there is improvement in Bs v6
